install.py

```diff
@@ -10,7 +10,19 @@
 import platform
 import warnings
 import datetime
+import importlib
 
+def install_and_import(package):
+    import importlib
+    try:
+        importlib.import_module(package)
+    except ImportError:
+        import pip
+        pip.main(['install', package])
```

Comment:
```suggestion
		subprocess.check_call('{0} -m pip install {1}'.format(sys.executable, package))
```
Start it as a subprocess instead. 

**References:**
1. https://pip.pypa.io/en/stable/user_guide/?highlight=_internal#using-pip-from-your-program
2. https://stackoverflow.com/a/47059613/10309266